CONTRIBUTING.md

```diff
@@ -10,6 +10,10 @@ Before submitting a Pull Request, please ensure the following:
 2. Make sure all tests are passing
 3. Only have one commit with a relevant message (you can rebase and squash all your commits into a single one)
 4. Check that you're up to date by rebasing your branch with our `develop` branch
+5. Do not forget to check that the code meets our standards:
+   - `mix format` (if your version is >= `1.6`)
```

Comment:
I don't think we need to specify the `(if your version is >= `1.6`)` since it's supposed the case in development given the `.tool-versions`

Reply:
Hm. Yes. But in case contributor does not use Asdf ?
As the CI covers a fairly large matrix's version. I'm not convinced that "forcing" the use of Asdf is necessary?

Reply:
Agreed but forcing Elixir `>= 1.6` for contirbuting to development is not an issue imho.